katsdptelstate/aio/redis.py

```diff
@@ -34,6 +34,9 @@
 
 logger = logging.getLogger(__name__)
 _QueueItem = Tuple[bytes, Optional[bytes]]
+# Note: this must be valid UTF-8, because aioredis decodes it if it needs to
+# reconnect to the server.
+_dummy_channel = b'\0katsdptelstate-internal0\001'
```

Comment:
Should this be `_DUMMY_CHANNEL` since it's a literal?

Reply:
Quite right - fixed! For some reason I thought about it when I wrote the code and convinced myself that it should be lower-case, but I have no idea why.